samples/sensor/ms5837/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _ms5837-sample:
+
+MS5837 Sensor Sample
+####################
+
+Overview
+********
+
+Sample application that every 10 seconds retrieves the pressure and temperature
```

Comment:
This should be a complete sentence, so how about:

    This sample application retrieves the pressure and temperature
    from a MS5837 sensor every 10 seconds, and prints this
    information to the UART console.